app/views/projects/samples/index.turbo_stream.erb

```diff
@@ -12,6 +12,7 @@
                       pagy: @pagy
                     } %>
 
-<%= turbo_stream.update "project_samples_hidden_search_value" do %>
+<%= turbo_stream.update "project_samples_hidden_values" do %>
   <%= render Ransack::HiddenSortFieldComponent.new(@q) %>
+  <%= hidden_field_tag "metadata", params[:metadata] %>
```

Comment:
**nitpick (blocking):** I believe this can be removed from here as you moved the `metadata` field into the search form.

Reply:
Correct, removed in 2e86100b703a4c8e4f3e65c424d0cf393bf1d3d4